src/Message/HpsGatewayResponseValidation.php

```diff
@@ -50,51 +45,56 @@ public static function getException($responseCode, $responseText, $response)
         case '0':
             break;
         case '-2':
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 'Authentication Error. Please double check your service configuration', HpsExceptionCodes::AUTHENTICATION_ERROR
             );
             break;
         case '3':
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 $responseText, HpsExceptionCodes::INVALID_ORIGINAL_TRANSACTION
             );
             break;
         case '5':
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 $responseText, HpsExceptionCodes::NO_OPEN_BATCH
             );
             break;
         case '12':
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 'Invalid CPC data', HpsExceptionCodes::INVALID_CPC_DATA
             );
             break;
         case '27':
         case '34':
         case '26':
         case '13':
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 'Invalid card data', HpsExceptionCodes::INVALID_CARD_DATA
             );
             break;
         case '14':
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 'The card number is not valid', HpsExceptionCodes::INVALID_NUMBER
             );
             break;
         case '30':
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 'Gateway timed out', HpsExceptionCodes::GATEWAY_ERROR
             );
             break;
         case '1':
         default:
-            $e = new InvalidResponseException(
+            $e = self::formResponseException(
                 $responseText, HpsExceptionCodes::UNKNOWN_GATEWAY_ERROR
             );
         }
 
         return $e;
     }
+    
+    public static function formResponseException($responseException, $exceptionCode = '')
+    {
+        return $responseException;
+    }
```

Comment:
This isn't creating the exception object nor combining/adding the message and the exception code together.